src/components/ImageUpload.tsx

```diff
@@ -1,65 +1,98 @@
+//       size: null;
+//       type: string;
+//     };
+//     releaseDate: string;
+//     title: string;
+//     yourRole: string;
+//   }>;
+// }
+
+export const ImageUpload = () => {
+  const PUBLIC_CLOUDINARY_CLOUD_NAME = "projectnewm",
+    PUBLIC_CLOUDINARY_UPLOAD_PRESET = "rvktckuk";
+
+  const [uploadedFilePublicId, setUploadedFilePublicId] = useState<string>();
+  const onDrop = useCallback(acceptedFiles => {
+    const url = `https://api.cloudinary.com/v1_1/${PUBLIC_CLOUDINARY_CLOUD_NAME}/upload`;
+
+    acceptedFiles.forEach(async (acceptedFile: File) => {
+      const formData = new FormData();
+      formData.append("file", acceptedFile);
+      formData.append("upload_preset", PUBLIC_CLOUDINARY_UPLOAD_PRESET);
+
+      const response = await fetch(url, {
+        body: formData,
+        method: "post",
+      });
+
+      const data = await response.json();
+      setUploadedFilePublicId(data.public_id);
+    });
+  }, []);
 
-export const ImageUpload = (props: ImageUploadProps) => {
-  const { setFieldValue } = props;
   const { getRootProps, getInputProps, isDragActive } = useDropzone({
     accept: "image/*",
-    onDrop: acceptedFiles => {
-      setFieldValue("files", acceptedFiles);
-    },
+    multiple: false,
+    onDrop,
   });
   return (
     <Box>
-      { }
       <div { ...getRootProps({ className: "dropzone" }) }>
-        <input { ...getInputProps() } />
-        <StyledPaperInput
-          sx={ {
-            width: "326px",
-            height: "38px",
-          } }
-        >
-          { " " }
-          Drag & Drop{ " " }
-        </StyledPaperInput>
-
-        <Typography variant="body1" align="center" color="primary">
-          OR
-        </Typography>
-        { isDragActive ? <p>Drop the files here ...</p> : <p>Drag n drop some files here, or click to select files</p> }
+        { !uploadedFilePublicId ? (
+          <div>
+            <input { ...getInputProps() } />
+            <StyledPaperInput
+              sx= { {
+                height: !isDragActive ? "38px" : "158px",
+                width: "webkit-fill-available",
+              } }
+            >
+              { isDragActive ? "Drop That Art" : " Drag & Drop" }
+            </StyledPaperInput>
+            { !isDragActive ? (
+              <Box sx={ { alignItems: "center", display: "flex", flexDirection: "column", justifyContent: "center" } }>
+                <Typography variant="body1" align="center" color="primary" marginTop="30px" marginBottom="27px">
+                  OR
+                </Typography>
+                <StyledFilledButton sx={ { width: "164px" } }>Upload Image</StyledFilledButton>
+              </Box>
+            ) : (
+              ""
+            ) }
+          </div>
+        ) : (
+          <Box sx={ { alignItems: "center", display: "flex", justifyContent: "center" } }>
+            <Image
+              cloudName={ PUBLIC_CLOUDINARY_CLOUD_NAME }
+              publicId={ uploadedFilePublicId + ".png" }
```

Comment:
will they always be `png`s?

Reply:
Yes, unless you think otherwise. The image had a white background when I uploaded a jpeg, but converting it to a png allowed it to have a transparent background.